README.md

```diff
@@ -1,4 +1,34 @@
 # teslausb
+## Armbian orange pi zero 2 setup *tweaks
+
+apt pacakages:
+
+* xfsprogs
+* dos2unix
+* hping3
+* libxxhash0
+* autofs
+* python3-pip
+* nginx
+* fcgiwrap
+* libnginx-mod-http-fancyindex
+* libfuse-dev
+* exfatprogs
+* sntp
```

Comment:
Install before using the teslausb setup script.

i just added it from the og teslausb repo since i had not come across it before and ran into problems on armbian specifically with not having packages before starting.

Reply:
Can you be more specific as to which problems you ran in to? As I said, these packages are installed by TeslaUSB during initial setup, and don't need to be installed manually beforehand.